src/world/ObjectMgr.h

```diff
@@ -270,6 +270,67 @@ enum AREATABLE_CATEGORY
     AREAC_SANCTUARY          = 6,
 };
 
+struct simpleEventScripts
+{
+    uint32 eventId;
+    uint32 function;
+    uint32 scripttype;
+    uint32 data_1;
+    uint32 data_2;
+    uint32 data_3;
+    uint32 data_4;
+    uint32 data_5;
+    uint32 x;
+    uint32 y;
+    uint32 z;
+    uint32 o;
+    uint32 delay;
+    uint32 nextevent;
+
+};
+
+enum ScriptCommands
```

Comment:
Please make these enums strongly typed. To do this, change
```
enum ScriptCommands
```
to
```
enum class ScriptCommands : uint8
```
And then reference members as
```
ScriptCommands::SCRIPT_COMMAND_TALK
ScriptCommands::SCRIPT_COMMAND_EMOTE
```